test/AutoRest.TestServer.Tests/Common/StringExtensionsTests.cs

```diff
@@ -79,6 +79,24 @@ public void ValidateSplitByCamelCase(string camelCase, string[] expected)
         [TestCase("_propertyName", true, "PropertyName")]
         [TestCase("_property_name", false, "propertyName")]
         [TestCase("_property_name", true, "PropertyName")]
+        [TestCase("_OS_profile", true, "OSProfile")]
+        [TestCase("_OS_profile", false, "osProfile")]
+        [TestCase("_OS_Profile", true, "OSProfile")]
+        [TestCase("_OS_Profile", false, "osProfile")]
+        [TestCase("OS", true, "OS")]
+        [TestCase("OS", false, "os")]
+        [TestCase("DNS", true, "DNS")]
+        [TestCase("DNS", false, "dns")]
+        [TestCase("OSProfile", true, "OSProfile")]
+        [TestCase("OSProfile", false, "osProfile")]
+        [TestCase("OSProfileId", true, "OSProfileId")]
+        [TestCase("OSProfileId", false, "osProfileId")]
+        [TestCase("VCenterIP", true, "VCenterIP")]
+        [TestCase("VCenterIP", false, "vCenterIP")]
+        [TestCase("D365IP", true, "D365IP")]
+        [TestCase("D365IP", false, "d365IP")]
+        [TestCase("P2SVpn", true, "P2SVpn")]
+        [TestCase("P2SVpn", false, "p2SVpn")]
```

Comment:
I'd expect this one to be `P2SVpn -> p2sVpn`.

Reply:
Well, this is unfortunate. This function `ToCleanName` does not have the ability to identify words, therefore we either always deal with digits to be separator of words, or always do not deal with digits to be separator of words.
If we do not treat digits as word separators, we will fail at the case `D365IP`.

Reply:
Please add a follow up item to address this since this happens today